modules/documentation/slides_documentation.md

```diff
@@ -0,0 +1,57 @@
+<!-- .slide: data-state="title" -->
+
+# Documentation
+
+---
+
+<!-- .slide: data-state="standard" -->
+
+## What is documentation?
+
+- Provides context for your work
+- Allows your collaborators and future you to understand what has been done and why
+
+---
+
+<!-- .slide: data-state="standard" -->
+
+## Why document software?
+
+Make your software reusable:
+ 
+- A user should be able to run your software in the same way as you do now 
+- A user should be able to install your software
+- A contributor should be able to add or improve code
+
+---
+
+<!-- .slide: data-state="standard" -->
+
+## Documentation types
+
+User documentation
+- Purpose: What does the software do?
+- How can it be used: provide examples
+
+Developer documentation
+- How can your software be modified or extended?
+
+Deployment documentation
+- Hardware and software requirements
+
+---
+
+<!-- .slide: data-state="standard" -->
+
+## Documentation formats
```

Comment:
[The Turing Way chapter about documentation (under construction](https://github.com/c-martinez/the-turing-way/pull/3/files?short_path=75ced25#diff-75ced25f81167559c20c1da1040f29f5111b5e2ced7df554fd5672c41792e1d6)) refers to 
[Divio Documentation System](https://docs.divio.com/documentation-system/), which mentions Tutorials, How-to guides, Explanations and refernce guides as different types. We could also use a the same framework here.